purchases/src/main/kotlin/com/revenuecat/purchases/customercenter/CustomerCenterConfigData.kt

```diff
@@ -0,0 +1,468 @@
+
+            @SerialName("google_subscription_manage")
+            GOOGLE_SUBSCRIPTION_MANAGE,
+
+            @SerialName("amazon_subscription_manage")
+            AMAZON_SUBSCRIPTION_MANAGE,
+
+            @SerialName("platform_mismatch")
+            PLATFORM_MISMATCH,
+            ;
+
+            val defaultValue: String
+                get() = when (this) {
+                    NO_THANKS -> "No, thanks"
+                    NO_SUBSCRIPTIONS_FOUND -> "No Subscriptions found"
+                    TRY_CHECK_RESTORE -> "We can try checking your Apple account for any previous purchases"
+                    RESTORE_PURCHASES -> "Restore purchases"
+                    CANCEL -> "Cancel"
+                    BILLING_CYCLE -> "Billing cycle"
+                    CURRENT_PRICE -> "Current price"
+                    EXPIRED -> "Expired"
+                    EXPIRES -> "Expires"
+                    NEXT_BILLING_DATE -> "Next billing date"
+                    REFUND_CANCELED -> "Refund canceled"
+                    REFUND_ERROR_GENERIC -> "An error occurred while processing the refund request. Please try again."
+                    REFUND_GRANTED -> "Refund granted successfully!"
+                    REFUND_STATUS -> "Refund status"
+                    SUB_EARLIEST_EXPIRATION -> "This is your subscription with the earliest expiration date."
+                    SUB_EARLIEST_RENEWAL -> "This is your subscription with the earliest billing date."
+                    SUB_EXPIRED -> "This subscription has expired."
+                    CONTACT_SUPPORT -> "Contact support"
+                    DEFAULT_BODY -> "Please describe your issue or question."
+                    DEFAULT_SUBJECT -> "Support Request"
+                    DISMISS -> "Dismiss"
+                    UPDATE_WARNING_TITLE -> "Update available"
+                    UPDATE_WARNING_DESCRIPTION ->
+                        "Downloading the latest version of the app may help solve the problem."
+                    UPDATE_WARNING_UPDATE -> "Update"
+                    UPDATE_WARNING_IGNORE -> "Continue"
+                    PLATFORM_MISMATCH -> "Platform mismatch"
+                    PLEASE_CONTACT_SUPPORT -> "Please contact support to manage your subscription."
+                    APPLE_SUBSCRIPTION_MANAGE ->
+                        "You can manage your subscription by using the App Store app on an Apple device."
+                    GOOGLE_SUBSCRIPTION_MANAGE ->
+                        "You can manage your subscription by using the Play Store app on an Android device"
+                    AMAZON_SUBSCRIPTION_MANAGE ->
+                        "You can manage your subscription in the Amazon Appstore app on an Amazon device."
+                }
+        }
+
+        fun commonLocalizedString(key: CommonLocalizedString): String {
+            return localizedStrings[key.name.lowercase()] ?: key.defaultValue
+        }
+
+        override fun equals(other: Any?): Boolean {
```

Comment:
Had to add all these equals because I'm not using data classes :(, I honestly don't know what's best, since we now need to remember to maintain them...

Reply:
Maybe we can just use data classes here? These are not _really_ public API classes. 

Data classes are primarily bad for binary compatibility. Any library depending on RevenueCat should not be calling `Purchases.getCustomerCenterConfigData()` anyway. (If we're worried about that, we could add a note to that function saying it's not guaranteed to be binary compatible.)

Reply:
Yeah... I think I might do that since maintaining all these equals is a bit annoying 👍